### PR TITLE
Typeable value on gradio.Slider

### PR DESCRIPTION
**Pull request by dawoodkhan82**
# Description
Allows slider value to be typeable.

https://user-images.githubusercontent.com/12725292/192082923-40c7f4dc-2e67-4c7c-aa0d-10a6cfa757d2.mov

Please include: 
* relevant motivation
* a summary of the change 
* which issue is fixed. 
* any additional dependencies that are required for this change.

Closes: #936

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by abidlabs — APPROVED** [review]

**Comment by abidlabs** [comment]
Very nice @dawoodkhan82! Tested and works well, not sure what's up with that test though

**Comment by dawoodkhan82** [comment]
Fixed tests, by updating test configs + updating kitchen sink Image input tests

**Review by pngwn — APPROVED** [review]
lgtm!

File: .github/workflows/ui.yml [hunk 1]
```diff
@@ -54,6 +54,7 @@ jobs:
           cache: pnpm
           cache-dependency-path: ui/pnpm-lock.yaml
       - run: pip install -r ../requirements.txt
+      - run: pip install gradio
```
Comment (dawoodkhan82) [inline .github/workflows/ui.yml:57]: @abidlabs added this in order to run the `create_demo_configs.py` file, which creates the configs for the integration tests. Confused how the tests were passing without this.

**Comment by siraxe** [comment]
Is it possible to enable/disable it ? or change width of the input field ?
Have a lot of small sliders and naming starts to overlap/distort the element

**Comment by abidlabs** [comment]
Hi @siraxe can you share a screenshot so we have an idea of how it looks? 

@dawoodkhan82 what do you think about adding a `typeable=True` parameter into the `Slider` that configures this?

**Comment by siraxe** [comment]
> 
Looks like this
https://user-images.githubusercontent.com/27487254/193601220-5d4ea46e-524b-4c5c-a667-f7b909ac2076.mp4
Maybe there's a way to override css options with js or something to at least change their width or yeah having `typeable=True` would be great as well )

**Pull request merged**